vendor/github.com/sirupsen/logrus/go.mod

```diff
@@ -8,3 +8,5 @@ require (
 	github.com/stretchr/testify v1.2.2
 	golang.org/x/sys v0.0.0-20190422165155-953cdadca894
 )
+
+go 1.13
```

Comment:
These lines keep getting added and removed. `dep ensure` removes them. Not sure why they get added back in, but they _are_ currently in this file: https://github.com/sirupsen/logrus/blob/d30efdb30d2e4f712eafaba4c33f952d084e6720/go.mod

Not a big deal but maybe it hints at a minor need to improve something somewhere with regards to vendored files.

Reply:
@nilsanderselde if you can work this out, please do.  Not blocking this PR.